index.ts

```diff
@@ -233,9 +243,17 @@ export default function viteNodeCGPlugin(pluginConfig: PluginConfig): Plugin {
             }
         },
 
-        configResolved(resolvedConfig: ResolvedConfig) {
+        async configResolved(resolvedConfig: ResolvedConfig) {
             // Capture resolved config for use in injectAssets
             config = resolvedConfig
+            // Check to see if one of the plugins is vite:react-refresh
+            if (resolvedConfig.plugins.find((plugin) => plugin.name === 'vite:react-refresh')) {
```

Comment:
`.find()` returns the found object but this is only used as a boolean. should use `.some()` instead.